app/controllers/holiday_entitlements_controller.rb

```diff
@@ -18,11 +18,11 @@ def edit
   end
 
   def create
-    @holiday_entitlement = HolidayEntitlement.new(holiday_entitlement_params)
+    @holiday_entitlement = @staff.holiday_entitlementsi.new(holiday_entitlement_params)
 
     respond_to do |format|
       if @holiday_entitlement.save
-        format.html { redirect_to holiday_entitlements_path, notice: 'Holiday entitlement was successfully created.' }
+        format.html { redirect_to admin_holiday_entitlements_path, notice: 'Holiday entitlement was successfully created.' }
```

Comment:
Line is too long. [124/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.